core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -661,6 +664,7 @@ ClientStream newSubstream(
       channel = builder.binlog.wrapChannel(channel);
     }
     this.interceptorChannel = ClientInterceptors.intercept(channel, interceptors);
+    this.transportFilters = builder.transportFilters;
```

Comment:
We need to make a copy so we know they aren't mutated. (Someone can add a filter after calling build(), then call build() a second time.)